### ISSUE TITLE
No more notification

### ISSUE THREAD

**Issue by Thatoo** [theAkito/webmon#50]
Last version installés from F-droid on Android 13
I don't get any notification anymore whereas everything was working well under previous version of Android.

**Comment by theAkito** [comment]
Greetings 👋🏻 

Thank you for this report.

Is it the same phone as before?

Did you try restarting the phone or force stopping the app and re-opening?

**Comment by Thatoo** [comment]
Yes, same phone, I did try to reboot and even uninstall and reinstall the app.
The problem is that Webmon doesn't request for notification autorisation and if I go in android settings > applications > Webmon, I can't tick autorisation for notification.

Best regards,

**Comment by theAkito** [comment]
I see. Probably some new Android thing. I'll look into it, when I will get the time.

**Comment by merlinuwe** [comment]
Thanks for your great app.

The same for me on Xiaomi Poco X3 Pro with Android 11 . I set it, also checked that notifactions are set on, rebootet, but I get no message.

![Image](https://github.com/user-attachments/assets/cedfebd3-c5ad-4163-a797-a58871983517)

**Comment by theAkito** [comment]
> The same for me on Xiaomi Poco X3 Pro with Android 11 . I set it, also checked that notifactions are set on, rebootet, but I get no message.

This is very difficult to replicate. For example, I used this app on the same phone and Android version *for years*, without any such issue.

Could you please add more technical specifications?

* Exact Android versions, including Kernel version.
* Which custom ROM you use & which additions are installed.
* Which Webmon version you use.
* Any other information, that might help. For example, might another app interfere with this one?
* Is the device rooted?

I need all this information, as this error is quite rare.

It would probably also help, if you could gather logs of the app, while it is not working, as expected.

Is there anything useful in the in-app log, when you experience the issue?

Did it never work in the first place or did it stop working at some point, after working fine for a while?

**Comment by merlinuwe** [comment]
Sure.

Webmon 2.9.0

No rooted device.

No custom ROM.

Where do I find the logs? (There is no menu entry for that or have I overseen this?) The folder `/Android/data` is empty.

I work with a pi-hole. (disable it doesn't change the issue)

I work with wg tunnel app 3.6.6 (disable it doesn't change the issue)

![Image](https://github.com/user-attachments/assets/8e84667d-116d-4bd3-a3ed-f26f072dce1a)

**Comment by theAkito** [comment]
> No rooted device.
> 
> No custom ROM.

Worst case, it might just be this. https://dontkillmyapp.com/



> Where do I find the logs? (There is no menu entry for that or have I overseen this?) The folder `/Android/data` is empty.

Sorry, forgot, that you specifically need to enable it. Go into Settings -> Advanced Settings -> Debug Log. App restarts. Now, re-go the same way and press "Show Log". Maybe you find something at first glance. Either way, finish with "Share Log" to save the log.

**Comment by merlinuwe** [comment]
Found the **solution**: 

Here you see, that it works. ;-)

![Image](https://github.com/user-attachments/assets/60488d4c-c8be-4afc-b175-a030c46462a9)

---

The solution:

![Image](https://github.com/user-attachments/assets/edeac63e-b65c-4ff6-852d-a617455df3f9)

---

The cause:

There was no hint like "enable autostart" in the settings, so I didn't thought about it:

![Image](https://github.com/user-attachments/assets/54e402e4-b59c-4d70-95d5-8f55420dad12)

---

It would be nice to have such a reminder or link, perhaps in one of the next releases?

Thank you very much for your fast response, and I'm lucky to got it running as expected. ;-)

**Comment by theAkito** [comment]
@Thatoo Did your issue get solved, too?

**Comment by Thatoo** [comment]
No, soory, it keeps saying "This app doesn't send notifications".
And I don't know where to find this "ernable auto strat" button.

**Comment by merlinuwe** [comment]
> No, soory, it keeps saying "This app doesn't send notifications". And I don't know where to find this "ernable auto strat" button.

Android settings -> 

![Image](https://github.com/user-attachments/assets/b6466288-f0ed-4632-971d-7903ff2c0a49)

->

![Image](https://github.com/user-attachments/assets/5147d60e-d96d-4c48-b6fb-570bb9107f84)
